### PR TITLE
Advertise about preparedb usage

### PR DESCRIPTION
**Pull request by Yajo**
This helpful thing wasn't documented.

### CODE REVIEW

File: docs/daily-usage.md [hunk 1]
```diff
@@ -93,10 +93,16 @@ Start Odoo with:
 invoke start
 ```
 
+Apply some common configurations for devel to work fine with:
```
Comment (joao-p-marques) [inline docs/daily-usage.md:96]:
```suggestion
You can also apply some common configurations to help you setup the devel environment (like setting the `report.url` paramenter):
```

File: docs/daily-usage.md [hunk 2]
```diff
@@ -93,10 +93,16 @@ Start Odoo with:
...
+
+```bash
+invoke preparedb
+```
+
 All of the above in one shot:
 
 ```bash
-invoke img-pull img-build --pull git-aggregate resetdb start
+invoke img-pull img-build --pull git-aggregate resetdb preparedb start
```
Comment (joao-p-marques) [inline docs/daily-usage.md:105]: This is not needed, as preparedb is executed by default when resetting the DB, unless you specify `--no-populate`
```suggestion
invoke img-pull img-build --pull git-aggregate resetdb start
```
And, as we are changing :smile: , this is the default also:
```suggestion
invoke img-pull img-build git-aggregate resetdb start
```

File: docs/daily-usage.md [hunk 3]
```diff
@@ -93,10 +93,17 @@ Start Odoo with:
...
+
+```bash
+invoke preparedb
+```
+
 All of the above in one shot:
 
 ```bash
-invoke img-pull img-build --pull git-aggregate resetdb start
+invoke img-pull img-build git-aggregate resetdb start
```
Comment (Yajo) [inline docs/daily-usage.md:106]: Revert this, please, it's not the same. `inv img-build --pull` pulls the dockerfile dependencies.

Comment (joao-p-marques) [reply docs/daily-usage.md:106]: Yes, but `--pull` is true by default: https://github.com/Tecnativa/doodba-copier-template/blob/stable/tasks_downstream.py#L387

Comment (Yajo) [reply docs/daily-usage.md:106]: 🤦🏼‍♂️ true

**Comment by Yajo** [comment]
@joao-p-marques approve to merge.

**Review by joao-p-marques — APPROVED** [review]

**Pull request merged**